sale_order_revision/tests/test_sale_order_revision.py

```diff
@@ -43,35 +41,34 @@ def test_order_revision(self):
 
         # Check the previous revision of the sale order
         revision_1 = sale_order_1.current_revision_id
-        self.assertEqual(sale_order_1.state, 'cancel')
-        self.assertFalse(sale_order_1.active)
+        self.assertEqual(sale_order_1.state, "cancel")
+        # self.assertFalse(sale_order_1.active)
```

Comment:
Please remove this if unused

Reply:
I tested this on my local machine. It's working with the uncommented line as well. No change in code coverage.
As there is no change for sale.order active I think this can be dropped.